package.json

```diff
@@ -65,6 +65,7 @@
     "jed": "^1.1.1",
     "js-beautify": "^1.6.12",
     "lodash": "^4.17.4",
+    "qs": "^6.4.0",
```

Comment:
We're already using the shimmed [native `querystring`](https://nodejs.org/api/querystring.html) in `actions.js`. Do we need this?